cmd/etl_worker/etl_worker.go

```diff
@@ -197,7 +197,12 @@ func worker(w http.ResponseWriter, r *http.Request) {
 	p := parser.NewParser(dataType, ins)
 	tsk := task.NewTask(fn, tr, p)
 
-	err = tsk.ProcessAllTests()
+	files, err := tsk.ProcessAllTests()
+
+	// Count the files processed per-host-module per-weekday.
+	metrics.FileCount.WithLabelValues(
+		data.Host+"-"+data.Pod+"-"+data.Experiment,
```

Comment:
I'd suggest having host and pod be separate labels.  It shouldn't impact the metric footprint very much, and would allow us to aggregate easily across hosts.

Up to you.  Also fine to leave a TODO.